ansible/controller_files/playbook.yml

```diff
@@ -0,0 +1,68 @@
+- name: Deploy ToDo app
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: secret_key
+      private: true
+      prompt: Enter the Flask secret key
+      salt_size: 8
```

Comment:
Minor but from the [Ansible docs](https://docs.ansible.com/ansible/latest/user_guide/playbooks_prompts.html#encrypting-values-supplied-by-vars-prompt) it looks like this is just the default; so worth considering whether highlighting its existence (👍 ) is worth the extra lines/comprehension required (👎 )
> By default Ansible generates a salt of size 8.

For the record I don't feel strongly either way, but I would usually skip explicitly setting default values